coderd/database/migrations/000155_workspace_agent_script.up.sql

```diff
@@ -0,0 +1,36 @@
+BEGIN;
+CREATE TABLE workspace_agent_log_sources (
+	workspace_agent_id uuid NOT NULL,
+	id uuid NOT NULL,
+	created_at timestamptz NOT NULL,
+	display_name varchar(127) NOT NULL,
+	icon text NOT NULL,
+	PRIMARY KEY (workspace_agent_id, id)
+);
+
+CREATE TABLE workspace_agent_scripts (
+	workspace_agent_id uuid NOT NULL,
+	log_source_id uuid NOT NULL,
```

Comment:
Any reason we wouldn't want to `REFERENCES workspace_agent_log_sources(id)` (potentially with `ON DELETE CASCADE`)? We could consider dropping agent ID in favor of the reference, since it's available in the other table.